lib/data/demos.dart

```diff
@@ -442,6 +443,22 @@ List<GalleryDemo> materialDemos(GalleryLocalizations localizations) {
       ],
       category: GalleryDemoCategory.material,
     ),
+    GalleryDemo(
+      title: localizations.demoDividerTitle,
+      icon: GalleryIcons.divider,
+      slug: 'divider',
+      subtitle: localizations.demoDividerSubtitle,
+      configurations: [
+        GalleryDemoConfiguration(
```

Comment:
Can you make a second configuration with `VerticalDivider`:

https://api.flutter.dev/flutter/material/VerticalDivider-class.html